### ISSUE TITLE
Will there be a new version of Numberbatch?

### ISSUE THREAD

**Issue by lsorber** [commonsense/conceptnet-numberbatch#52]
It's been a while since the last update, and I was wondering if there are any new versions planned. Thanks for the great work so far!

**Comment by simonsays1980** [comment]
Agree. There are also a lot domain-specific words missing from Wikipedia like e.g. webdav, kubernetes, etc. Are there any plans to consider corpuses with domain specific words as a basis for numberbatch? 'webdav' for example is included in the GloVe vectors. I can't find it in numberbatch.

**Comment by HenkPoley** [comment]
There seems to be a numberbatch-20180108 floating around.

**Comment by lsorber** [comment]
Looks like there's a new version out [1], kudos to the authors for the new release!

[1] https://github.com/commonsense/conceptnet-numberbatch

**Comment by michael-robbins** [comment]
What's the chances of a newer version now that COVID/etc terms are running rampant?
